### PR TITLE
Add "How did you hear about us" to Enterprise form

### PR DESCRIPTION
**Pull request by ericholscher**
This also updates all the copy on the Enterprise plan page to sell it a bit more effectively.

Fixes https://github.com/readthedocs/meta/issues/142

----
📚 Documentation preview 📚: https://readthedocs-about--297.org.readthedocs.build/

### CODE REVIEW

**Review by humitos — APPROVED** [review]
Looks 👍🏼 to me. I left some small suggestions.

File: content/pages/enterprise.html [hunk 1]
```diff
@@ -94,36 +92,42 @@
...
-          <h2 class="ui center aligned dividing header">Enterprise plan inquiry</h2>
+          <h2 class="ui center aligned dividing header">Talk with a sales rep about an Enterprise plan</h2>
           <p>
-            Our Enterprise plans have a number of additional features that are not available in our standard plans.
             Let us know what you're looking for and we'll work with you to find the right solution.
           </p>
           <div class="field">
             <label>Name</label>
-            <input type="text" name="name">
+            <input type="text" name="name" placeholder="Jane Smith">
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply content/pages/enterprise.html:101]: Yea, seems outside the scope here, but I think placeholders are generally pretty light.

File: content/pages/enterprise.html [hunk 2]
```diff
@@ -94,36 +92,42 @@
...
           </div>
           <div class="field">
             <label>Company email address</label>
-            <input type="email" name="email">
+            <input type="email" name="email" placeholder="name@company.com">
+          </div>
+          <div class="field">
+            <label for="referral">How did you hear about us?</label>
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply content/pages/enterprise.html:108]: Nope. It just passes everything through.

File: content/pages/enterprise.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (humitos) [reply content/pages/enterprise.html:101]: I just realized that when the input has focus, the placeholder is a little more visible 👍🏼

**Pull request merged**